recipes/cppcheck/all/patches/0001-cmake-source-dir.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/oss-fuzz/CMakeLists.txt b/oss-fuzz/CMakeLists.txt
+index b9a3e195ea8cf64fd9c078c5102d82a25b2435ca..41bcdccf4e536f08aa5e450320a715d7df8b5b84 100644
+--- a/oss-fuzz/CMakeLists.txt
++++ b/oss-fuzz/CMakeLists.txt
+@@ -5,7 +5,7 @@ if (ENABLE_OSS_FUZZ AND CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+             $<TARGET_OBJECTS:simplecpp_objs_sanitized>
+             $<TARGET_OBJECTS:tinyxml_objs_sanitized>
+             $<TARGET_OBJECTS:lib_objs_sanitized>)
+-    target_include_directories(fuzz-client PRIVATE ${CMAKE_SOURCE_DIR}/lib ${CMAKE_SOURCE_DIR}/externals/simplecpp ${CMAKE_SOURCE_DIR}/externals/tinyxml ${CMAKE_SOURCE_DIR}/externals)
```

Comment:
I like to recommend keeping patches small. It appears that some of these are not relevant to building.

Would you mind trying to cut of the ones which are not a hard requirement?

Reply:
Reduced it to dmake, thank you for all your comments!